js/examples/openai/compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   phoenix:
-    image: arizephoenix/phoenix:latest
+    image: arizephoenix/phoenix:nightly
```

Comment:
latest doesn't have annotations right now

Reply:
We could pin to this for now? https://github.com/Arize-ai/phoenix/releases/tag/arize-phoenix-v4.22.0